src/backend/core/order/views.py

```diff
@@ -66,6 +70,316 @@ def get(self, request):
         return super().get(request)
 
 
+class TestView(APIView):
```

Comment:
Do we actually need this view? 🙂

Reply:
Probably not, must be some leftover from one bugfix. Removed.

Reply:
Cool
Great job btw. 🙂